lib/buildingsync/model_articulation/building.rb

```diff
@@ -40,7 +40,7 @@
 require_relative '../helpers/stat_file'
 module BuildingSync
   class Building < SpatialElement
-    include OsLib_ModelGeneration
+    include OsLib_ModelGenerationBRICR
```

Comment:
@TMaile could we make a da branch of openstudio-extension-gem, push the modified version of this file there, and have you work off that branch of the extension-gem for a while?  We can follow that pattern for other repos we want to move stuff from this gem to.